internal/pkg/cli/env_init.go

```diff
@@ -48,11 +51,19 @@ var (
 	errNamedProfilesNotFound = fmt.Errorf("no named AWS profiles found, run %s first please", color.HighlightCode("aws configure"))
 )
 
+type tempCredsVars struct {
+	AccessKeyID     string
+	SecretAccessKey string
+	SessionToken    string
+}
+
 type initEnvVars struct {
 	*GlobalOpts
-	EnvName      string // Name of the environment.
-	EnvProfile   string // AWS profile used to create an environment.
-	IsProduction bool   // Marks the environment as "production" to create it with additional guardrails.
+	EnvName      string        // Name of the environment.
+	EnvProfile   string        // AWS profile used to create an environment.
+	IsProduction bool          // Marks the environment as "production" to create it with additional guardrails.
+	TempCreds    tempCredsVars // Temporary credentials to initialize the environment. Mutually exclusive with the AWS profile.
```

Comment:
Should there be something here about only one of `EnvProfile` or `TempCreds` can be used or does it matter?

Reply:
There is a comment for TempCreds: `Mutually exclusive with the AWS profile.` and `Validate()` will throw an error if both are set. 

Or am I misunderstanding/missing something?

Reply:
Bah i meant to put this comment on line 63. I think elsewhere our phrasing has typically been "only one of X or Y can be used" but the mutually exclusive thing in this line is fine.